### DISCUSSION TITLE
Workflow Integrations

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#345] [Website Discussions]
# Workflow Integrations

DaVinci Resolve Studio now supports Workflow Integration Plugins to be loaded and communicate with Resolve.

https://resolve.cafe/developers/workflowintegrations/

**Comment by Franz78it** [comment]
Hi, is it possible to add jquery in the referenced js files?
 and use it to communicate with an external service via ajax call?

Thanks for the kind replies
